pkg/transponder/configure.go

```diff
@@ -82,6 +82,13 @@ func Configure(cmd *cobra.Command, args []string) error {
 		return errors.New("failed to list connections")
 	}
 
+	fmt.Println("Initiating data discovery")
```

Comment:
There is a chance that we might not be doing data discovery if the creds are wrong.  Ideally we want the customer to do the ls command again to check if the connection has turned active.  Is there a way to communicate that?

If not, lets commit this :D